src/tools/genpolicy/src/registry.rs

```diff
@@ -64,8 +65,37 @@ pub struct DockerRootfs {
 pub struct ImageLayer {
     pub diff_id: String,
     pub verity_hash: String,
+    pub passwd: String,
 }
 
+/// A single record in a Unix passwd file.
+#[derive(Debug)]
+struct PasswdRecord {
+    pub user: String,
+    #[allow(dead_code)]
+    pub validate: bool,
```

Comment:
Nit: I believe the second field has three interpretations: login disabled, password login, passwordless. Doesn't matter much for this implementation, though.